src/resources/views/province/show.blade.php

```diff
@@ -0,0 +1,82 @@
+@extends('appshell::layouts.private')
+
+@section('title')
+    {{ $province->name }}
+@stop
+
+@push('page-actions')
+    <x-appshell::standard-actions
+        :model="$province"
+        :name="$province->name"
+        :edit-url="route('appshell.province.edit', [$country, $province])"
+        :delete-url="route('appshell.province.destroy', [$country, $province])"
+    />
+@endpush
+
+@section('content')
+    <div class="row mb-3">
+        <div class="col">
+            <x-appshell::card-with-icon icon="plan" type="success">
+                {{ $province->name }}
+
+                <x-slot:subtitle>
+                    PROVINCE
+                </x-slot:subtitle>
```

Comment:
Let the theme/css decide whether it is going to be all uppercase + always translate UI texts
```suggestion
                <x-slot:subtitle>{{ __('Province') }}</x-slot:subtitle>
```